testutil/garbage/garbage_test.go

```diff
@@ -0,0 +1,74 @@
+package garbage
+
+import (
+	"bytes"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/ipld/go-ipld-prime"
+	"github.com/ipld/go-ipld-prime/codec/dagcbor"
+	"github.com/ipld/go-ipld-prime/datamodel"
+	"github.com/ipld/go-ipld-prime/node/basicnode"
+)
+
+func TestGarbageProducesAllKinds(t *testing.T) {
+	kindCount := make(map[datamodel.Kind]int)
+	seed := time.Now().Unix()
+	fmt.Printf("randomness seed: %v\n", seed)
```

Comment:
nit: use `t.Logf` for all of these, so the output is attached to the test result

Reply:
👍